src/components/CreateMemorandum/UploadMemorandum.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-card flat id="upload-memorandum">
+
+    <!-- Summary Section -->
+    <div id="upload-memorandum-summary" v-if="isSummary">
```

Comment:
Since this is a v-card, could v-card-text etc be used for some of these blocks? Or would it conflict with the other, similar components?

Reply:
Absolutely, although i'm not even sure what the summary view looks like for these components.

Reply:
OK, next person to work in these files can fix it.